docs/project/changelog.md

```diff
@@ -20,6 +20,8 @@ myst:
 - {{ Enhancement }} Improved logging in `pyodide-build` with rich.
   {pr}`3442`
 
+- {{ Enhancement}} Added `pyodide-build` support for building a list of packages from a requirements.txt file. Also auto-output a list of chosen dependencies in the same format when building a package and dependencies. This enables repeatable builds of packages. {pr}`3469`
```

Comment:
Maybe the second sentence could be a bit rephrased, as  `manually output` by opposition to `auto-output` doesn't make sense for software :)
For instance
```
This command also writes the list of dependencies detected when building packages to `requirements.txt` for better reproducibility.
```

Also repeatable ->  reproducible.